### DISCUSSION TITLE
Every recipe duplicated after import

### DISCUSSION THREAD

**Discussion by robsug90** [TandoorRecipes/recipes#3162] [Q&A]
I have about 250 recipes. During the import process (which took about a day weirdly) I've noticed every single recipe is duplicated. Are there any functions or quick ways to resolve this without going one by one?

Thank you.

**Comment by vabene1111** [comment]
Sadly there is no mass delete feature that would resolve this easily. What importer did you use, I never heard of it taking a day or duplicating recipes.

What kind of sever are you running on?

**Reply by robsug90** [reply]
Seems like a functionality to reverse or remove the import would be useful.
I imported Paprika recipes using the built in importer.
In terms of the speed, I think that was me setting the container's clock speed way too low.
